.azure-pipelines/official.yml

```diff
@@ -0,0 +1,27 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 8 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+  always: true
+
+stages:
+- stage: Build
+  pool: sonicbld
+
+  jobs:
+  - template: build-template.yml
+    parameters:
+      platform: broadcom
+      platform_short: brcm
+      cache_mode: rcache
```

Comment:
wcache building has Unit test issue. It is not azure pipeline issue.

Reply:
what unit test issue?

Reply:
official build should be wcache?

Reply:
AssertionError: assert 'broadcom' == 'x86_64-mlnx_msn2700-r0'
https://dev.azure.com/sonicswitch/build/_build/results?buildId=2599&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96

Reply:
I think official build should be wcache. I am not sure. Because we used an independent job to wcache in Jenkins.